.github/workflows/build-docsite.yaml

```diff
@@ -0,0 +1,34 @@
+name: Build the Ansible docsite
+
+on:
+  push:
+    branches:
+      - 'main'
+  workflow_dispatch:
+
+jobs:
+  build-docsite:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out the repository
+      uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.10"
+    - name: Install requirements
+      run: |
+        python -m pip install --upgrade pip
```

Comment:
hmm. I was just trying it out and it seems this is still needed. or do you mean that we should include pip in the requirements?

Reply:
Yes, that's just a hint, pip prints out if the current version is not the last available. You can suppress it by setting the `PIP_NO_PIP_VERSION_CHECK` env var to `1`. This can be done on the step level.

Reply:
FWIW, I tend to set all these vars in my boilerplate to make all the console output in GHA jobs nice: https://github.com/cherrypy/cheroot/blob/00c3cc3/.github/workflows/ci-cd.yml#L58-L67.

Reply:
very nice. I think I'll go with the var to disable the version check and look at the other boilerplate in a follow up PR. thanks for sharing that.

Reply:
FTR the var corresponds to a CLI argument. You can also pass `--disable-pip-version-check` if it's just one command below. But the boilerplate on the workflow level could be useful when used in bigger projects.